Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1511,7 +1511,7 @@ NOTE: Restart is required for the setting to take effect.</value>
     <comment>{0}: Variable Name</comment>
   </data>
   <data name="MoveCloserToUsage_TargetIsInOtherNonStandardModule" xml:space="preserve">
-    <value>'{0}' is defined neither in the component component it is used not in a standard module. </value>
+    <value>'{0}' is defined neither in the component component it is used nor in a standard module. </value>
```

Comment:
What's a `component component`?

Reply:
:facepalm:

Reply:
Actually, reading it again, it's still awkward constructed. 

What about this:
```
'{0}' is neither defined nor used in any non-standard modules
```

Reply:
I wouldn't not wonder if there might not be a way to make that not a bit less, not, ...or is it just me?